class-components/src/components/Floyt/Flyout.type.ts

```diff
@@ -0,0 +1,8 @@
+export interface People {
```

Comment:
You have **4 (!)** interfaces `People `across the project... If it's all related to one entity, please have only **1** interface `People `and remove the excessive ones. If some of them refer to a different entity, please rename the interfaces to better reflect their meaning.

 In any case, always make sure that all the exported interfaces, consts, functions always have a different name to reduce bugs in the code.